
uint64 timestamp	# Timestamp when camera was triggered
uint32 seq		# Image sequence

